tests/backstop/config.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
 			'--disable-gpu',
 			'--disable-setuid-sandbox',
 			'--no-zygote',
+			'--accept-lang=en_US',
```

Comment:
This should use a hyphen instead, as defined in the AC. I've confirmed this is correct in https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/chrome_switches.cc;l=23-25;drc=31405cbb988e1fd01c75b7a85178dac53623cbf9

```suggestion
			'--accept-lang=en-US',
```